scripts/src/services/applications.ts

```diff
@@ -0,0 +1,66 @@
+import { Application, AppWhitelists } from "../models/applications";
+import * as jsonwebtoken from "jsonwebtoken";
+import { getLogger } from "../logging";
+const logger = getLogger();
+
+type JWTClaims = {
+	iss: string; // issuer
+	exp: number; // expiration
+	iat: number; // issued at
+	sub: string; // subject
+};
+
+type JWTContent = {
+	header: {
+		typ: string;
+		alg: string;
+		key: string;
+	};
+	payload: JWTClaims & {
+		// custom claims
+		user_id: string;
+		api_key: string;
```

Comment:
add api_key to JWT claims. the api_key and app_id(issuer) must match the values in our DB. see `validateApiKey`